squirrels/_dashboards.py

```
@@ -141,14 +141,14 @@ class DashboardsIO:
 
     @classmethod
     def load_files(
-        cls, logger: u.Logger, base_path: str, auth_type: AuthType = AuthType.OPTIONAL, project_configurables: dict[str, Any] = {}
+        cls, logger: u.Logger, project_path: str, auth_type: AuthType = AuthType.OPTIONAL, project_configurables: dict[str, Any] = {}
```

Ruff (0.14.13)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)